### PR TITLE
Add a changelog

### PR DESCRIPTION
**Pull request by idanarye**
Since I'm planning on adding a PR with some breaking changes, it might be a good idea to have a place to document these changes in.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+### Fixed
+- Resolve jitter.
+
+## 0.2.0 - 2022-07-31
+### Changed
+- Update to bevy 0.8.
+
+## 0.1.5 - 2022-07-19
+### Added
+- Globaltransform to bundles.
```
Comment (PROMETHIA-27) [inline CHANGELOG.md]:
```suggestion
- GlobalTransform to bundles.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Starship example.
+- Preset constructors.
+
+### Changed
+- Tweak upright force to feel better.
+- Renam CCBundle to FPSBundle.
```
Comment (PROMETHIA-27) [inline CHANGELOG.md]:
```suggestion
- Rename CCBundle to FPSBundle.
```

**Review by PROMETHIA-27 — APPROVED** [review]
With some typo fixes LGTM!

**Comment by idanarye** [comment]
Applied your suggestions

**Pull request merged**